backend/src/models/Trace.ts

```diff
@@ -27,6 +27,7 @@ export class TraceModel {
   eventType!: string;
 
   @prop({ required: true })
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Remove?

Reply:
thats there to disable the linter warning.